src/model/gateway.rs

```diff
@@ -728,3 +730,316 @@ pub struct ActivityTimestamps {
     pub end: Option<u64>,
     pub start: Option<u64>,
 }
+
+
+/// [Gateway Intents] will limit the events your bot will receive via the gateway.
+/// By default, all intents except [Privileged Intents] are selected.
+///
+/// # What are Intents
+///
+/// A [gateway intent] sets the types of gateway events
+/// (e.g. member joins, guild integrations, guild emoji updates, ...) the
+/// bot shall receive. Carefully picking the needed intents greatly helps
+/// the bot to scale, as less intents will result in less events to be
+/// received via the network from Discord and less processing needed for
+/// handling the data.
+///
+/// # Privileged Intents
+///
+/// The intents [`GatewayIntents::GUILD_PRESENCES`] and [`GatewayIntents::GUILD_MEMBERS`]
+/// are [Privileged Intents]. They need to be enabled in the
+/// *developer portal*.
+///
+/// **Note**:
+/// Once the bot is in 100 guilds or more, [the bot must be verified] in
+/// order to use privileged intents.
+///
+/// [gateway intent]: https://discord.com/developers/docs/topics/gateway#privileged-intents
+/// [Privileged Intents]: https://discord.com/developers/docs/topics/gateway#privileged-intents
+/// [the bot must be verified]: https://support.discord.com/hc/en-us/articles/360040720412-Bot-Verification-and-Data-Whitelisting
+/// [`GatewayIntents::GuildPresences`]: serenity::client::bridge::gateway::GatewayIntents::GUILD_PRESENCES
+/// [`GatewayIntents::GuildMembers`]: serenity::client::bridge::gateway::GatewayIntents::GUILD_MEMBERS
+#[derive(Copy, PartialEq, Eq, Clone, PartialOrd, Ord, Hash)]
+pub struct GatewayIntents {
```

Comment:
Changing the location of a type is a major breaking change, which is forbidden on the `current` branch. You will have to retarget your PR to the `next` branch.